app/helpers/hosts_helper.rb

```diff
@@ -407,7 +407,8 @@ def allocation_text_f(f)
           :name => 'allocation_radio_btn',
           :class => (label == active) ? 'btn btn-default active' : 'btn btn-default',
           :onclick => "tfm.computeResource.libvirt.allocationSwitcher(this, '#{label}');",
-          :data => { :toggle => 'button' }
+          :data => { :toggle => 'button' },
+          :id => (label == 'Full') ? 'btnFull' : nil
```

Comment:
I'd rather do less generic id like "btnAllocationFull". Also, `nil` is not a good id at all, we aim to have ids for all elements for the future, can you set something like "btnAllocationNonFull"?

Reply:
should we have ids even if there is no later use of them?

Reply:
It's for QA so they can automate web browser bots. Anyway, I am merging, thanks.